examples/webserver_with_module.py

```diff
@@ -23,12 +24,14 @@ def __init__(self):
     async def message(self, _msg, conn):
         """ Respond to basic messages with a count of messages received. """
         self.count += 1
-        await conn.send_async({
-            "@type": self.type("message"),
-            "~l10n": {"locale": "en"},
-            "sent_time": utils.timestamp(),
-            "content": "{} message(s) received.".format(self.count)
-        })
+        await conn.send_async(
+            {
+                "@type": self.type("message"),
+                "~l10n": {"locale": "en"},
+                "sent_time": utils.timestamp(),
+                "content": "{} message(s) received.".format(self.count),
```

Comment:
why not use f strings?
```suggestion
                "content": f"{self.count} message(s) received.",
```

Reply:
I agree, fstrings are more concise. I'll follow up with a PR that switches all instances of format to fstrings soon.